<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rebuild_win</class>
 <widget class="QDialog" name="rebuild_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>226</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit User</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	font: auto &quot;Arial&quot;;
	font-size: 14px;
font-weight:bold;

}




#rebuild_win{
	background-color: #ffffff;
	border: 2px solid rgb(77, 77, 77);
	border-radius: 10px;
	
}



QPushButton{
	border: none;
	min-height: 40px;
	border-radius: 5px;

	
}
/*
QPushButton:hover{

	background-color:rgb(22, 38, 76);

}

QPushButton:disabled{
	color: #808080;
	background-color:rgb(210, 210, 210);

}
*/

/*********************************************/
QLabel{
	color: rgb(50,50,50);
color: #404040;
}


/*********************************************/


/*******************************************/
QLineEdit
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 16px;
}



/*******************************************/


QGroupBox{
	font-weight: bold;
}

QLineEdit:hover{
	border: 1px solid black;
	background: #e4f0fa;
}

QLineEdit:focus {
	color: rgb(6, 76, 130);
	background: #ffffff;
	selection-background-color: rgb(255, 204, 75);
}

QSpinBox:hover{
	background: #e4f0fa;
}


/*********************************************/
QComboBox
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 14px;
}

QComboBox:enabled{
color: rgb(50, 50, 50);
}

QComboBox:disabled
{
	border:2px solid rgb(210, 210, 210);
	color:(210, 210, 210);

}

QComboBox::down-arrow
{   
	image: url(:/assets/Assets/icons/icons8-arrow-48.png);
	width: 15px;
    height: 15px;

}

QComboBox::drop-down
{
	 background-color: rgb(238, 134, 44);
	 min-width: 30px;
}

QComboBox::drop-down:disabled 
{
	 background-color: rgb(210, 210, 210);
	 min-width: 30px;
}

QComboBox QAbstractItemView {
    border: none;
    selection-background-color: rgb(6, 76, 130);
	selection-color: rgb(6, 76, 130);


}

QComboBox QAbstractItemView {
    border: none;
}

QComboBox QAbstractItemView::item {
    border: none;
	height:30px;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	padding:3px;
	border:none;
}

QPushButton:hover{

border: 2px solid #404040;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Assets/Assets.qrc">
         <normaloff>:/assets/icons/icons8-close-50.png</normaloff>:/assets/icons/icons8-close-50.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>You edit some standards, You should rebuild your database </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="converting_progressbar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="massage_lbl">
       <property name="styleSheet">
        <string notr="true">color:rgb(41, 147, 108);</string>
       </property>
       <property name="text">
        <string>Rebuilde Success</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="rebuild_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{

background-color: rgb(58, 209, 154);
color:#ffffff;
max-width: 200px;
}


QPushButton:hover{

background-color: rgb(49, 177, 130);
color:#ffffff;
}

QPushButton:disabled{
background-color: rgb(220, 220, 220);
color:#909090;
max-width: 200px;
}</string>
         </property>
         <property name="text">
          <string>Rebuild</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets/Assets.qrc"/>
 </resources>
 <connections/>
</ui>
